egs2/rats/spk1/local/data_prep.py

```diff
@@ -0,0 +1,84 @@
+import argparse
+import glob
+import os
+import pickle as pk
+import sys
+
+
+def main(args):
+    src = args.src
+    dst = args.dst
+
+    head_dir = "/".join(args.src.split("/")[:-3])
+    f_info = head_dir + "/docs/source_file_info.tab"
+    src2spk = {}
+    for line in open(f_info, "r").readlines()[1:]:
+        src_id, lng, spk, t, part = line.strip().split()
+        src2spk[src_id] = spk
+    pk.dump(src2spk, open(dst + "src2spk", "wb"))
+
+    spk2utt = {}
+    utt2spk = []
+    wav_list = []
+
+    fs = glob.glob(os.path.join(src, "audio/*/*/*.flac"))
```

Comment:
I think you're assuming that the RATs data is already downloaded as exists in a specific directory. 

I believe there are cases (especially when the data needs to be purchased) where we don't support the download in `data.sh`. 
However, in this case, please add a few lines of comments on how you need the dataset or filetree to be prepared).